setup.py

```diff
@@ -29,8 +29,8 @@ def get_static(name, condition=None):
 # scripts to be added to the $PATH
 scripts = get_static("scripts", condition="'.' in x")
 
-with open("VERSION", 'r') as versionfile:
-	version = versionfile.read().strip()
+with open("looper/_version.py", 'r') as versionfile:
```

Comment:
is this really the best practice? Open a python file and parse it to extract the version? There must be a better way...